webapp/tests/karma/js/enketo/widgets/db-object-widget.spec.ts

```diff
@@ -0,0 +1,86 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+describe('Enketo: DB Object Widget', () => {
+
+  let dbObjectWidget;
+  let model;
+  let nodeFn;
+  let getCurrentForm;
+  let getElements;
+  let setVal;
+
+  beforeEach(() => {
+    dbObjectWidget = require('../../../../../src/js/enketo/widgets/db-object-widget');
+    getElements = sinon.stub();
+    setVal = sinon.stub();
+    nodeFn = sinon.stub().returns({ getElements, setVal });
+    model = { node: nodeFn };
+    getCurrentForm = sinon.stub();
+    window.CHTCore = { Enketo: { getCurrentForm } };
```

Comment:
Minor, but `window.CHTCore` is always being reset in each of the following tests.  I guess we can either remove this here, or just remove the dup calls from the tests....